save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/VulnerabilitiesFiltersRow.kt

```diff
@@ -97,7 +103,7 @@ val vulnerabilitiesFiltersRow: FC<VulnerabilitiesFiltersProps> = FC { props ->
                         getUrlForOptionsFetch = { prefix ->
                             "$apiUrl/organizations/get/by-prefix?prefix=$prefix&pageSize=$DROPDOWN_OPTIONS_AMOUNT"
                         }
-                        placeholder = "Organization..."
+                        placeholder = "${t("Organization")}..."
```

Comment:
as a suggestion (@akuleshov7):
we can refactor `fun t(String) -> String` to
```kotlin
val String.t: String
    get() = TODO()
```
and then `"${t("Organization")}..."` can be refactored to `"${"Organization".t}..."`

Reply:
I am not sure how to implement it so that `t` is val with custom getter, while t should be returned from `useTranslation` hook

Reply:
Done

Reply:
@akuleshov7 please look at it